.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "typescript.preferences.importModuleSpecifier": "relative"
+  "[typescript]": {
+    "typescript.preferences.importModuleSpecifier": "relative"
+  },
 }
+
```

Comment:
No need for a double new line 😅 It was a single one that was missing.

Reply:
Sorry, the IDE formatter was playing games with me 😄